components/server/package.json

```diff
@@ -16,11 +15,11 @@
     "clean:node": "rimraf node_modules",
     "purge": "yarn clean && yarn clean:node && yarn run rimraf yarn.lock",
     "lint": "yarn tslint -p .",
-    "test": ". $(leeway run components/gitpod-db:db-test-env) && TS_NODE_FILES=true mocha --opts mocha.opts './**/*.spec.ts' --exclude './node_modules/**'",
-    "spicedb-testing": "if netstat -tuln | grep ':50051 '; then echo '`spicedb serve-testing` is already running.'; else spicedb serve-testing --load-configs /workspace/gitpod/install/installer/pkg/components/spicedb/data/schema.yaml; fi",
-    "stop-spicedb-testing": "PID=$(lsof -t -i:50051); if [ -z \"$PID\" ]; then echo '`spicedb serve-testing` is not running'; else kill -INT $PID && echo 'Stopped `spicedb serve-testing`'; fi",
-    "db-test": "r(){ . $(leeway run components/gitpod-db:db-test-env); yarn spicedb-testing & yarn db-test-run; yarn stop-spicedb-testing; };r",
-    "db-test-run": "TS_NODE_FILES=true mocha --opts mocha.opts '**/*.spec.db.ts' --exclude './node_modules/**'",
+    "test": ". $(leeway run components/gitpod-db:db-test-env) && yarn test:unit && (yarn start-spicedb & (yarn test:db && yarn stop-spicedb))",
```

Comment:
I assume this works because `yarn test` is the default `command.test` for yarn packages? :thinking:

Reply:
Yes, not cool that this is hidden within leeway. I always need to look up the leeway code in order to understand the semantics.

Reply:
:100: Would it make sense to explicitly add that to the package? config.commands.test is the path I think.